fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingScrubMissing.java

```diff
@@ -235,7 +229,7 @@ public IndexingScrubMissing(@Nonnull final IndexingCommon common,
                         logWarningCounter --;
                         LOGGER.warn(KeyValueLogMessage.build("Scrubber: missing index entry",
                                         LogMessageKeys.KEY, rec.getPrimaryKey().toString(),
-                                        LogMessageKeys.INDEX_KEY, missingKey.toString())
+                                        LogMessageKeys.INDEX_KEY, missingIndexesKeys.toString())
```

Comment:
Since the `filter()` wasn't really filtering, you'll want to strip out the `null` values in the list before logging.